torch_frame/nn/models/excelformer.py

```diff
@@ -37,30 +39,55 @@ def feature_mixup(
         y (Tensor): The target.
         num_classes (int): Number of classes.
         beta (float): The concentration parameter of the Beta distribution.
+        mixup_type (str): The mixup methods.
+        mi_scores (Tensor): Mutual information scores for FEAT-MIX.
 
     Returns:
         x_mixedup (Tensor): The mixedup numerical feature.
         y_mixedup (Tensor): Transformed target of size
             :obj:`[batch_size, num_classes]`
     """
     assert num_classes > 0
+    assert mixup_type in ['ordinary', 'feature', 'hidden']
```

Comment:
where is 'none'?

Reply:
Thank you! I have fixed it with only  `none` & `feature` & `hidden` options.